plonky2x/src/frontend/vars/variable.rs

```diff
@@ -107,3 +109,142 @@ impl<L: PlonkParameters<D>, const D: usize> One<L, D> for Variable {
         Variable(builder.api.one())
     }
 }
+
+impl<L: PlonkParameters<D>, const D: usize> LessThan<L, D> for Variable {
+    type Output = BoolVariable;
+
+    fn lt(self, rhs: Variable, builder: &mut CircuitBuilder<L, D>) -> Self::Output {
+        // FIXME: Check for overflows and edge cases
+        const NUM_LIMBS: usize = 64;
```

Comment:
Shouldn't this depend on the Field? I think in particular `F::order()` should work and return to you the order of the field, and you can get `NUM_LIMBS` from that. 

Another option is that here you panic if `2**{NUM_LIMBS} != log_2(order)` just as a runtime check since I think it's quite unlikely that people will not use Goldilocks (there might be an off by one in the equation above but you get the point).

Reply:
fixed here [4d760f8](https://github.com/succinctlabs/succinctx/pull/152/commits/4d760f819628fff3fb5158ae8b6938ce6e607bfc)